2-mandatory/9-cheap-diner.js

```diff
@@ -19,7 +19,14 @@ chosenMeal(emptyArray)
 Should give the answer "Nothing :("
 **/
 
-function chooseMeal(mealArray) {}
+function chooseMeal(mealArray) {
+    return mealArray.length > 1
+      ? mealArray.sort((a, b) => a.price - b.price)[1]["name"]
+      : mealArray.length === 1
+      ? mealArray[0]["name"]
+      : "Nothing :(";
```

Comment:
ternarys can be nested, and this is fine, however I'd caution against nesting ternary operators like this as it can get a little messy/hard to read sometimes keeping track of the conditions that have (or have not) been met. In general, use a ternary if it's a simple `if(...) return x, else(...) return y` , else I'd stick with multiple `if` statements (more of a stylistic choice based on personal experience, this is also fine)